plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1246,6 +1246,8 @@ preferences.section.bedrock.description=On this page you can tweak the setting r
 preferences.section.notifications=Notifications
 preferences.section.notifications.description=On this page you can change settings regarding notifications in MCreator.
 preferences.ui.language=Interface language
+preferences.ui.usemacOSMenuBar=Use the default macOS menu bar
```

Comment:
Shouldn't it be `MacOS`? It would look better in my opinion...

Reply:
I said about it before

Reply:
macOS is the correct spelling, max you should know this